### DISCUSSION TITLE
How to use ConsoleLogger when importing at runtime

### DISCUSSION THREAD

**Discussion by Djokix98** [atteneder/glTFast#480] [Q&A]
Hello, I am trying to load the zipped gltf model at runtime and I made a custom unzipper and the IDownloadProvider class that should help with the textures in the file. I am using the success = await gltf.LoadGltfJson() however the loading never ends without any errors. So i wanted to check the logs in order to figure out what is exactly happening and i checked the ConsoleLogger and assigned it to the GltfImport like so > gltf = new GltfImport(logger: new ConsoleLogger());. However, this doesnt really print anything. Then i tried to make a custom one using ICodeLogger that calls Debug.Log() in every method, and assigned that one > gltf = new GltfImport(logger: new MyLogger());, however still nothing is happening. 
How do i use the logger correctly to print out steps and things that are happening when loading the gltf? 
Thanks in advance!

**Comment by atteneder** [comment]
Hi,

you're somehow relying that there are calls to the `ILogger` that give you hints to where the problem lies. That very well might not be the case. I recommend using a debugger and breakpoints in your code to break down the problem. It's a little harder with async code (stepping over await might not work), but imho it's still a lot more productive than trying to fire a bunch of logs.

hth
